src/reflect/scala/reflect/internal/tpe/FindMembers.scala

```diff
@@ -285,4 +285,21 @@ trait FindMembers {
       initBaseClasses.head.newOverloaded(tpe, members)
     }
   }
+
+  private[scala] final class HasMember(tpe: Type, name: Name, excludedFlags: Long, requiredFlags: Long, stableOnly: Boolean)
+    extends FindMemberBase[Symbol](tpe, name, excludedFlags, requiredFlags) {
+    private[this] var _result: Symbol = NoSymbol
+
+    override protected def result: Symbol = _result
+
+    protected def shortCircuit(sym: Symbol): Boolean = {
+      _result = sym
+      true // prevents call to addMemberIfNew
```

Comment:
AFAICT this will only find the first member with `name`, which doesn't seem like what we want.